cmd/appliance/force_disable_controller.go

```diff
@@ -220,27 +220,26 @@ func forceDisableControllerRunE(opts cmdOpts, args []string) error {
 			}
 			for _, ctrl := range controllers {
 				if arg == ctrl.GetHostname() {
-					hostnameArgs = append(hostnameArgs, ctrl.GetHostname())
+					idList = append(idList, ctrl.GetId())
 					continue ARG_LOOP
 				}
 			}
 			for _, ctrl := range offline {
 				if arg == ctrl.GetHostname() {
-					hostnameArgs = append(hostnameArgs, ctrl.GetHostname())
+					idList = append(idList, ctrl.GetId())
 					continue ARG_LOOP
 				}
 			}
 			log.WithField("arg", arg).Info("No Controller found with provided hostname")
 		}
 		// automatically add offline controllers to disable list, but only if the user has actively selected a valid controller first
-		if len(hostnameArgs) > 0 {
+		if len(idList) > 0 {
 			for _, ctrl := range offline {
-				hostnameArgs = util.AppendIfMissing(hostnameArgs, ctrl.GetHostname())
+				idList = util.AppendIfMissing(idList, ctrl.GetId())
 			}
 		}
```

Comment:
I wonder if there is a bug when we compute the `offline` appliances that would automatically add non-healthy appliances. 

in `FilterAvailable` 
https://github.com/appgate/sdpctl/blob/4fc40b114ffa84afcf73b238ea92523acc627b3b/pkg/appliance/functions.go#L124

Reply:
Hmmm maybe not. This looks correct.
https://github.com/appgate/sdpctl/blob/4fc40b114ffa84afcf73b238ea92523acc627b3b/pkg/appliance/functions.go#L795